docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+
+= Configuring a run configuration
+
+You can configure a number of separate run configurations to run against each of your projects. Each configuration is saved automatically as you work.
```

Comment:
" a number of separate" - How about "multiple"?
"each of your projects" > "each project"?
"Each configuration is saved automatically as you work." > "The run configuration is automatically saved."